### PR TITLE
Upgrade built-in admin plugin to Kinto Admin 1.4.0

### PR DESCRIPTION
**Pull request by leplatrem**
r? @Natim 

Weird, I had to do some stuff by hand...

The `npm run dist` command did not exist.

version substitution in `__init__.py` file: 
```
sed -i "s/ version=\".*\"/ version=\"$(tmp/bin/kinto-admin --version)\"/g" kinto/plugins/admin/__init__.py
/bin/sh: 1: tmp/bin/kinto-admin: not found
```

And when I tested it, the fonts were not loading. I removed the url prefixes in the CSS file.

### CODE REVIEW

**Comment by Natim** [comment]
You shouldn't use the build:gh-pages command.
If dist doesn't exist anymore there it is probably a bug of the kinto-admin that removed it.
The `KINTO_ADMIN_PREFIX` is supposed to do the CSS job, also the kinto-admin command is supposed to exists which proves that something went wrong when you ran the command.

**Comment by Natim** [comment]
Apparently the new procedure is there: https://github.com/Kinto/kinto-admin/#installation
I will update the Makefile accordingly.

File: Makefile [hunk 1]
```diff
@@ -68,14 +69,11 @@ build-requirements:
...
-	git clone https://github.com/Kinto/kinto-admin.git tmp
-	cd tmp; git checkout $$(git tag | tail -n 1); npm install
-	cd tmp; KINTO_ADMIN_PUBLIC_PATH=/v1/admin/ npm run dist
-	cp -fr tmp/dist/* kinto/plugins/admin/static/
-	sed -i "s/ version=\".*\"/ version=\"$$(tmp/bin/kinto-admin --version)\"/g" kinto/plugins/admin/__init__.py
-	rm -fr tmp
+	rm -fr kinto/plugins/admin/www
+	rm -fr kinto/plugins/admin/node_modules
+	cd kinto/plugins/admin/; npm install && npm run build
+	mv kinto/plugins/admin/build kinto/plugins/admin/www
```
Comment (Natim) [inline Makefile]: Maybe we can keep build instead off www here.

**Comment by n1k0** [comment]
FYI kinto-admin v1.4.1 [has been released](https://github.com/Kinto/kinto-admin/releases/tag/v1.4.1).

File: kinto/plugins/admin/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# kinto-admin plugin setup
+
+Update the kinto-admin version you want to use in `package.json`, then:
+
+```
+$ npm i
+$ npm run build
+$ mv build static
```
Comment (Natim) [inline kinto/plugins/admin/README.md]: We don't need that anymore.

File: kinto/plugins/admin/__init__.py [hunk 3]
```diff
@@ -7,11 +7,11 @@ def includeme(config):
     # Expose capability.
     config.add_api_capability(
         "admin",
-        version="1.3.0",
+        version="1.4.0",
```
Comment (Natim) [inline kinto/plugins/admin/__init__.py]: 1.4.1

File: kinto/plugins/admin/public/index.html [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Kinto Admin</title>
```
Comment (Natim) [inline kinto/plugins/admin/public/index.html]: Kinto Web Administration

File: Makefile [hunk 5]
```diff
@@ -68,14 +69,10 @@ build-requirements:
...
-	rm -fr kinto/plugins/admin/static/*
-	rm -fr tmp
-	git clone https://github.com/Kinto/kinto-admin.git tmp
-	cd tmp; git checkout $$(git tag | tail -n 1); npm install
-	cd tmp; KINTO_ADMIN_PUBLIC_PATH=/v1/admin/ npm run dist
-	cp -fr tmp/dist/* kinto/plugins/admin/static/
-	sed -i "s/ version=\".*\"/ version=\"$$(tmp/bin/kinto-admin --version)\"/g" kinto/plugins/admin/__init__.py
-	rm -fr tmp
+	rm -fr kinto/plugins/admin/build
+	rm -fr kinto/plugins/admin/node_modules
```
Comment (leplatrem) [inline Makefile]: Why do we delete this folder each time?

Comment (Natim) [inline Makefile]: It was to install the last version of the dependencies. In anycase I don't like to have a node_modules folder in a Python software tree 🗡

**Pull request merged**